### PR TITLE
Make the flip timer work with 3 digits for days display

### PR DESCRIPTION
**Pull request by vladd08**

### CODE REVIEW

File: js/jquery.flipTimer.js [hunk 1]
```diff
@@ -141,9 +141,16 @@
...
+        } 
+        else if (subject.classList.contains('days') && _this.days > 99) {
+          var output = [];
+          var daysString = _this.days.toString();
+          for (var i = 0, len = daysString.length; i < len; i += 1) {
+            output.push(+daysString.charAt(i));
+          }
+          number_array = output;
+        }
+        else {
```
[the remark this answers is not in the document]
Comment (vladd08) [reply js/jquery.flipTimer.js]: My pleasure, Andew!
First of all, thanks for getting in touch. I did not expect it to be honest.
As for my PR, this code was not meant to be some final result towards the specifed goal, but rather someting quick that worked for me (and might work for others as well). I just did a quick fix for this to work as it met my needs at the moment :).
Anyhow, I'm really glad to apply your suggested changes, but I can not commit to a timeline. As soon as I get some time, I will look into it.

File: js/jquery.flipTimer.js [hunk 2]
```diff
@@ -58,7 +58,7 @@
     minutes: false,
     hours: false,
     days: false,
-    date: (new Date()).toDateString(),
+    date: 'January 1, 2013 08:30:30',
```
[the remark this answers is not in the document]
Comment (vladd08) [reply js/jquery.flipTimer.js]: Sure thing.

File: js/jquery.flipTimer.js [hunk 3]
```diff
@@ -216,7 +227,7 @@
         }
 
         // if timer runs out stop the timer
-        if ((_this.days > 99) || (_this.days == 99 && _this.hours == 23 && _this.minutes == 59 && _this.seconds == 59)) {
+        if ((_this.days > 999) || (_this.days == 999 && _this.hours == 23 && _this.minutes == 59 && _this.seconds == 59)) {
           clearInterval(_this.timer);
```
[the remark this answers is not in the document]
Comment (vladd08) [reply js/jquery.flipTimer.js]: Sure, I'll see what can be done 👍

[the remark this answers is not in the document]
Comment (vladd08) [reply js/jquery.flipTimer.js]: Actually I disagree on this one. I think the timer still needs to be stopped at some point. If you know / see some subtle stuff around, be my guest to see this through.

**Comment by vladd08** [comment]
Great news, @andrewjtait! I'll fix the broken test(s) later today.

**Comment by andrewjtait** [comment]
Thanks again, @vladd08 🎉

**Comment by vladd08** [comment]
My pleasure, @andrewjtait ! 🎉

**Pull request merged**